backend/src/app.ts

```diff
@@ -115,6 +115,9 @@ const poll = async () => {
     network = await SI.networkStats();
     try {
         cpuTemp = await SI.cpuTemperature();
+        if (!cpuTemp.cores.length && cpuTemp.max > 0) {
```

Comment:
I'm wondering if there is a better way to check if the `cpuTemp.max` is valid in this condition.

The issue is that a temperature of zero degrees, or even negative degrees is technically a perfectly valid temperature.

However, even the module we're using sets the default to -1°C, so we may not have a choice in this matter. If the module can't get the `max` value, it'll still return -1°C which is sadly a valid temperature.

Let's just hope for now that no one using Doppler on a Raspberry Pi is experimenting with LN2 cooling, or lives in the Artic Circle.